tools/binlog/deploy.md

```diff
@@ -0,0 +1,488 @@
+---
+title: TiDB-Binlog Cluster 部署
+category: tools
+---
+
+
+# TiDB-Binlog 部署
+
+## 使用 TiDB-Ansible 部署 TiDB-Binlog
+
+### 第 1 步：下载 TiDB-Ansible
+
+1. 以 TiDB 用户登录中控机并进入 `/home/tidb` 目录。以下为 TiDB-Ansible 分支与 TiDB 版本的对应关系，版本选择可咨询官方 info@pingcap.com。
+
+    | TiDB-Ansible 分支 | TiDB 版本 | 备注 |
+    | ---------------- | --------- | --- |
+    | release-2.0-new-binlog | 2.0 版本 | 最新 2.0 稳定版本，可用于生产环境。 |
+    | release-2.1 | 2.1 版本 | 最新 2.1 稳定版本，可用于生产环境（建议）。 |
+    | master | master 版本 | 包含最新特性，每日更新。 |
+
+2. 使用以下命令从 GitHub [TiDB-Ansible 项目](https://github.com/pingcap/tidb-ansible)上下载 TiDB-Ansible 相应分支，默认的文件夹名称为 `tidb-ansible`。
+
+    - 下载 2.0 版本：
+
+        ```bash
+        $ git clone -b release-2.0-new-binlog https://github.com/pingcap/tidb-ansible.git
+        ```
+
+    - 下载 2.1 版本：
+
+        ```bash
+        $ git clone -b release-2.1 https://github.com/pingcap/tidb-ansible.git
+        ```
+
+    - 下载 master 版本：
+
+        ```bash
+        $ git clone https://github.com/pingcap/tidb-ansible.git
+        ```
+
+### 第 2 步：部署 Pump
+
+1. 修改 tidb-ansible/inventory.ini 文件
+
+    1. 设置 `enable_binlog = True`，表示 TiDB 集群开启 binlog。
+
+        ```ini
+        ## binlog trigger
+        enable_binlog = True
+        ```
+
+    2. 为 `pump_servers` 主机组添加部署机器 IP。
+
+        ```ini
+        ## Binlog Part
+        [pump_servers]
+        172.16.10.72
+        172.16.10.73
+        172.16.10.74
+        ```
+
+        默认 Pump 保留 5 天数据，如需修改可修改 `tidb-ansible/conf/pump.yml` 文件中 `gc` 变量值，并取消注释，如修改为 7。
+
+        ```yaml
+        global:
+          # an integer value to control the expiry date of the binlog data, which indicates for how long (in days) the binlog data would be stored
+          # must be bigger than 0
+          gc: 7
+        ```
+
+        请确保部署目录有足够空间存储 binlog，详见：[部署目录调整](../../op-guide/ansible-deployment.md#部署目录调整)，也可为 Pump 设置单独的部署目录。
+
+        ```ini
+        ## Binlog Part
+        [pump_servers]
+        pump1 ansible_host=172.16.10.72 deploy_dir=/data1/pump
+        pump2 ansible_host=172.16.10.73 deploy_dir=/data1/pump
+        pump3 ansible_host=172.16.10.74 deploy_dir=/data1/pump
+        ```
+
+2. 部署并启动含 Pump 组件的 TiDB 集群
+
+    参照上文配置完 `inventory.ini` 文件后，从以下两种方式中选择一种进行部署。
+    
+    **方式一**：在已有的 TiDB 集群上增加 Pump 组件，需按以下步骤逐步进行。
+    
+    1. 部署 pump_servers 和 node_exporters
+    
+        ```
+        ansible-playbook deploy.yml -l ${pump1_ip}, ${pump2_ip}, [${alias1_name}, ${alias2_name}]
+        ```
+        
+    2. 启动 pump_servers
+    
+        ```
+        ansible-playbook start.yml --tags=pump
+        ```
+        
+    3. 更新并重启 tidb_servers
+    
+        ```
+        ansible-playbook rolling_update.yml --tags=tidb
+        ```
+        
+    4. 更新监控信息
+    
+        ```
+        ansible-playbook rolling_update_monitor.yml --tags=prometheus
+        ```
+
+    **方式二**：从零开始部署含 Pump 组件的 TiDB 集群
+    
+    使用 Ansible 部署 TiDB 集群，方法参考 [TiDB Ansible 部署方案](../op-guide/ansible-deployment.md)。
```

Comment:
update link

Reply:
Yep, should update the link to '../../op-guide/ansible-deployment.md'

Reply:
update in https://github.com/pingcap/docs-cn/pull/1264/commits/57ea74626bd14c447b9ad7e4e6fac9a242464d46